tests/test_apis/test_onnx2openvino.py

```diff
@@ -121,3 +121,15 @@ def test_get_input_info_from_cfg():
     input_info = get_input_info_from_cfg(deploy_cfg)
     assert input_info == expected_input_info, \
         'The expected value of \'input_info\' does not match the received one.'
+
+    # The case where the input name in 'onnx_config'
+    # is different from 'backend_config'.
+    onnx_config_input_name = 'image'
```

Comment:
could the input name for testing be randomly generated?

Reply:
Added random name generation in tests `test_onnx2openvino` and `test_get_input_info_from_cfg`.